src/features/instance/ProcessNavigationContext.tsx

```diff
@@ -104,6 +116,10 @@ function useProcessNext() {
   return { perform, error: utils.error };
 }
 
+function appUnlocksOnPDFFailure({ altinnNugetVersion }: ApplicationMetadata) {
+  return !altinnNugetVersion || isAtLeastVersion({ actualVersion: altinnNugetVersion, minimumVersion: '8.1.0.115' });
```

Comment:
Bare dobbeltsjekker at logikken er riktig: Enten at `altinnNugetVersion` ikke er satt, eller at den er minst versjon `8.1.0.115`?

Reply:
Heh, tenkte jeg ikke på en gang, jeg bare kopierte blindt fra et annet sted 😅 

Jeg tror eneste grunnen for at jeg la det inn er at den er nullable, men det har ingenting å si fordi hvis den ikke er definert, så funker ikke appen i det hele tatt og vi viser error-page. Det betyr basically at nuget-versjonene er lavere enn v8 som vi ikke støtter.

```ts
isValidVersion:
  !!data.altinnNugetVersion &&
  isAtLeastVersion({
    actualVersion: data.altinnNugetVersion,
    minimumVersion: MINIMUM_APPLICATION_VERSION.build,
}),

```